packages/appstore/src/components/modals/wallet-modal/wallet-modal-header.tsx

```diff
@@ -93,16 +92,9 @@ const WalletModalHeader = ({
         return { icon, ...size };
     }, [currency, getCurrencyIconSize, is_dark, is_demo]);
 
-    const getBackgroundName = React.useCallback(() => {
-        if (is_demo) {
-            return 'demo';
-        }
-        return currency.toLowerCase();
-    }, [is_demo, currency]);
-
     return (
-        <GradientBackground {...getWalletModalBackgrounds(getBackgroundName())}>
-            {is_demo && <Watermark image={`url(${is_dark ? DemoDark : DemoLight})`} opacity={is_dark ? 0.32 : 0.24} />}
+        <GradientBackground color='#FBDDDD'>
```

Comment:
@heorhi-deriv `#FBDDDD` is use as default one just for testing purposes and will be replaced with proper one in the future.
Watermark will be also uncommented in the future.